task/thermal/src/bsp/sidecar_bc.rs

```diff
@@ -86,14 +86,14 @@ impl Bsp {
         // controller and fan index:
         //
         // System Index    Controller     Fan           MAX31790 Fan (Datasheet)
-        //     0            East           NNE           2 (3)
-        //     1            East           SNE           3 (4)
-        //     2            East           Northeast     0 (1)
-        //     3            East           Southeast     1 (2)
-        //     4            West           Northwest     2 (3)
-        //     5            West           Southwest     3 (4)
-        //     6            West           NNW           0 (1)
-        //     7            West           SNW           1 (2)
+        //     0            East           ENE           2 (3)
+        //     1            East           ESE           3 (4)
```

Comment:
How did you determine which rotor is which in the assembly?

Reply:
The drawing we got from SD notes an inlet vs outlet fan. In our system the inlet fan is more "north" and connects to pwm0/tach0, while the outlet fan is more "south" and connects to pwm1/tach1. How we then connect those signals to east/west controllers is not straightforward, as you noted in the issue.